src/pages/Overview/SchoolNav/SchoolNav.jsx

```diff
@@ -1,26 +1,88 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import './SchoolNav.css';
+import { Container } from 'react-bootstrap';
+import Signin from '../../../components/Signin';
+import { AuthProvider } from '../../../contexts/AuthContext';
+
+function LoginView(props) {
+  const { school } = props; // destructuring prop or else would get linter error
+  return (
+    <>
+      <h3>{school} is using a virtual textbook exchange!</h3>
+      <h5>To buy or sell textbooks, please login or create an account below.</h5>
+
+      <AuthProvider>
```

Comment:
i would put the provider around the routes that will use it. (aka similar to how we have `<Providers>` wrap about the routes in `src/App.jsx`

Reply:
That way we dont have to call this provider multiple times